src/app/action-items/action-items.component.ts

```diff
@@ -20,10 +26,12 @@ export class ActionItemsComponent implements OnInit, OnDestroy {
   dataSource: ActionItemMapped[];
   isLoadingActionItems = false;
   isLoadingActionItem = false;
-  actionItemId: string | undefined;
-  actionItem: ActionItem;
-  errorMessage: string | undefined;
-  justAddedItemId: string;
+  actionItemId?: string;
+  actionItem?: ActionItem;
```

Comment:
**SUGGESTION:** General note - please make sure that all fields have the proper modifiers (so if something is not used in template then it's private).